src/psmove.c

```diff
@@ -45,6 +45,15 @@
 #include <sys/stat.h>
 #include <math.h>
 #include <limits.h>
+#include <stdarg.h>
+
+#if defined(__linux) || defined(__APPLE__)
```

Comment:
Shouldn't that just check for "not Apple" to include locale.h for Linux and Win32? Apple is covered in the `#ifdef` below.

Reply:
Yep, and that actually broke Win32 builds, ALL platforms need `locale.h`, and macOS also needs `xlocale.h`. Fixed now.